src/search/renderDateFilterPair.js

```diff
@@ -26,6 +26,8 @@ function renderSingleDateFilter(intl, filterStruct, updateQuery, field, boundary
         updateQuery({ filters: deparseFilters(fs2) });
       }}
       useInput
+      usePortal
```

Comment:
Maybe you should fix the comment higher up the file that says "usePortal" doesn't fix this?